docs/metrics/customizing-the-sdk/Program.cs

```diff
@@ -29,19 +29,19 @@ public static void Main()
             .AddView(instrumentName: "MyCounter", name: "MyCounterRenamed")
 
             // Change Histogram boundaries using the Explicit Bucket Histogram aggregation.
-            .AddView(instrumentName: "MyHistogram", new ExplicitBucketHistogramConfiguration() { Boundaries = new double[] { 10, 20 } })
+            .AddView(instrumentName: "MyHistogram", new ExplicitBucketHistogramConfiguration() { Boundaries = [10, 20] })
```

Comment:
Just to highlight that we are creating double's array. It affects couple of other places.

```suggestion
            .AddView(instrumentName: "MyHistogram", new ExplicitBucketHistogramConfiguration() { Boundaries = [10.0, 20.0] })
```

Reply:
You need to fix other places.